frontend/src/views/admin/matching/styles.css

```diff
@@ -0,0 +1,517 @@
+    background-color: #d5d5d5;
+    transition: width 0.5s ease;
+}
+
+.status-sidebar.empty {
+    background-color: #d70000;
+}
+
+.status-sidebar.under {
+    background-color: #ffcc16;
+}
+
+.status-sidebar.matched {
+    background-color: #00c12b;
+}
+
+.status-sidebar.over {
+    background-color: #00e5f3;
+}
+
+.status-sidebar.selected {
+    width: 25px;
+}
+
+.position-row.over {
+    background: rgba(0, 229, 243, 0.15);
+}
+
+.position-row.matched {
+    background: rgba(0, 193, 43, 0.15);
+}
+
+.position-row > .position-title {
+    flex-grow: 1;
+    padding-left: 10px;
+    font-weight: 600;
+}
+
+.position-row > .position-hours-filled {
+    padding-left: 10px;
+    padding-right: 10px;
+    font-weight: 250;
+    opacity: 0.7;
+}
+
+.container-filler {
+    flex-grow: 1;
+}
+
+/* GRID VIEW */
+.grid-view-list {
+    display: flex;
+    flex-wrap: wrap;
+    /*overflow: scroll;*/
+    row-gap: 6px;
+    column-gap: 8px;
+}
+
+.grid-view-section:not(:first-child) {
+    margin-top: 10px;
+}
+
+.grid-view-section > h4 {
+    margin-bottom: 4px;
+}
+
+.applicant-grid-item {
+    display: flex;
+    width: 170px;
+    border-radius: 5px;
+    overflow: hidden;
+    align-items: center;
+    background-color: #fff;
+    filter: drop-shadow(2px 2px 2px #bbb);
+    margin-bottom: 5px;
+    cursor: pointer;
+
+    font-size: smaller;
+}
+
+.applicant-grid-item > * {
+    display: flex;
+}
+
+.applicant-status-sidebar {
+    width: 30px;
+    height: 100%;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
+
+    color: #fff;
+    background-color: #cfcfcf;
+}
+
+.applicant-status-sidebar.empty {
+    background-color: #d70000;
+}
+
+.applicant-status-sidebar.under {
+    background-color: #ffcc16;
+}
+
+.applicant-status-sidebar.matched {
+    background-color: #00c12b;
+}
+
+.applicant-status-sidebar.over {
+    background-color: #00e5f3;
+}
+
+.applicant-grid-main {
+    flex-direction: column;
+    margin-left: 5px;
+    flex-grow: 1;
+}
+
+.applicant-grid-main:nth-child(1) {
+    margin-top: 3px;
+}
+
+.applicant-grid-main:nth-child(2) {
+    margin-bottom: 3px;
+}
+
+.grid-row {
+    display: flex;
+    flex-direction: row;
+    column-gap: 5px;
+}
+
+.grid-detail-small {
+    min-width: 20px;
+    padding-left: 2px;
+    padding-right: 2px;
+    border-radius: 2px;
+    text-align: center;
+    font-weight: 500;
+    color: #868686;
+    background: #eeeeee;
+}
+
+.applicant-name {
+    width: 100px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.matching-course-header {
+    padding: 0 3px;
+    position: sticky;
+    top: 0;
+    background-color: white;
+    z-index: 10;
+}
+
+.matching-course-body {
+    padding: 0 3px;
+}
+
+.matching-course-body > .table-responsive {
+    font-size: smaller;
+}
+
+.applicant-status-divider {
+    width: 75%;
+    height: 1px;
+    background-color: #fff;
+}
+
+.applicant-status-value {
+    flex-grow: 1;
+}
+
+.no-highlight {
```

Comment:
Why do you need this class? It seems like it's meant to paper over other incorrect code.

Reply:
Overriding Bootstrap styling on toggle button groups to remove their highlighting/border/box-shadow. :(

Reply:
Okay. That's a good enough excuse.